client/config/karma.conf.js

```diff
@@ -76,7 +76,7 @@ module.exports = function(config) {
      * available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
      */
     browsers: [
-      'Firefox'
+      'Chrome'
     ],
```

Comment:
You had to manually define the location of Chromium. That's what is done in the .travis.yml now.